Cargo.toml

```diff
@@ -21,7 +21,7 @@ publish = false
 
 # See https://github.com/cgwalters/cargo-vendor-filterer
 [package.metadata.vendor-filter]
-platforms = ["x86_64-unknown-linux-gnu"]
+platforms = ["x86_64-unknown-linux-gnu", "s390x-unknown-linux-gnu"]
```

Comment:
Or actually, just something like `*-unknown-linux-gnu`. Does the filterer support that?